src/tests/tcmalloc_unittest.cc

```diff
@@ -1290,6 +1290,7 @@ static int RunAllTests(int argc, char** argv) {
     free(p1);
     VerifyDeleteHookWasCalled();
 
+#if HAVE_DECL_CFREE || TCMALLOC_ENABLE_LIBC_OVERRIDE
```

Comment:
Should this be `&&` ?

Reply:
Even if we don't have system cfree, we will get cfree alias if the libc override is active. So I think this is the correct condition.